app/templates/client/styles/_variables.scss

```diff
@@ -0,0 +1 @@
+$unit: 0.625rem;
```

Comment:
Is this still used anywhere?

Reply:
It is. Although I don't know the history of why the magic number exists as a base unit for things.